superx/information_extractor.py

```diff
@@ -161,14 +167,16 @@ def convert_unit_name(self, unit_in_hebrew):
             'ק"ג': ['קילו', 'ק"ג', 'קילו', 'קילוגרמים'],
             'גרם': ['גרם', 'גרמים'],
             'ליטר': ['ליטר', 'ליטרים', 'ליטר    '],
-            'מ"ל': ['מיליליטרים', 'מ"ל', 'מיליליטר']
-            }
+            'מ"ל': ['מיליליטרים', 'מ"ל', 'מיליליטר'],
+            'אין': ['יחידה', 'לא ידוע']
+        }
 
         for unit in unit_dict.keys():
-            if unit_in_hebrew in unit:
+            if unit_in_hebrew in unit_dict[unit]:
                 return unit
 
-        return 'unknown'
+        # as a default return the original unit
+        return unit_in_hebrew
```

Comment:
You may want to log this as well, so that you know that you need to look at it and maybe add it to the list.